ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -878,6 +878,7 @@ const PanelMain = fp.flow(
                       </label>
                       <Dropdown
                         id='runtime-autopause'
+                        appendTo='self'
```

Comment:
what was the problem here?  no visible change

Reply:
The default behavior of our version of PrimeReact's DropDown component is to add the dropdown's expansion panel to the end of the <body> tag. By using `appendTo='self`, the expansion panel is added inside of the component's wrapper tag. They look visually identical (as far as I can tell). The reason that I have been updating the drop downs to `appendTo='self` is that many of our e2e tests have xpath selectors that are looking for the panel inside the wrapper rather than at the end of the body tag. This has been the source of many of the e2e failures that we have seen since updating React 17 and PrimeReact.

Current State:
![Screen Shot 2023-02-10 at 3 39 04 PM](https://user-images.githubusercontent.com/24514630/218193525-662c7136-2d39-40f6-9236-a9100114475e.png)

Updated State:
![Screen Shot 2023-02-10 at 3 43 02 PM](https://user-images.githubusercontent.com/24514630/218194223-d3753502-fc12-4adf-b071-bfbcf6a6b186.png)